modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -352,14 +351,6 @@ private void onTableCreateInternal(@NotNull ConfigurationNotificationEvent<Table
                         return CompletableFuture.allOf(futures);
                     }
 
-                    @Override
-                    public @NotNull CompletableFuture<?> onRename(@NotNull String oldName, @NotNull String newName,
-                            @NotNull ConfigurationNotificationEvent<TableView> ctx) {
-                        // TODO: IGNITE-15485 Support table rename operation.
```

Comment:
This code had very important TODO, you should return it back

Reply:
Rollback